server/modules/resourceFactories/DynamoTableResource.js

```diff
@@ -91,6 +91,28 @@ class DynamoTableResource {
     });
   }
 
+  query(params) {
+    let self = this;
+    let request = {
+      TableName: this._tableName,
+      KeyConditionExpression: `${this._keyName} = :key`,
+      ExpressionAttributeValues: {
+        ":key": params.key
+      }
+    };
+
+    return this.client.query(request).promise().then(data => {
+      if (!data.Items) {
```

Comment:
again, I think we shouldn't do this. Unfortunately in JavaScript `0 == false == '' == []` and data.Items could be any of these here. `data.Items.length > 0` ? `... !== undefined`?